Aptoide/aptoide-client-v8
# [AN-1853] Added an empty state card when there are no posts in a timeline.

To test:
  Stores > ana2016 > timeline tab > followers > click dsgna/Qualquercoisa > timeline tab > should show the empty state list. This should happen on pull-to-refresh, on retry and on view created
